packages/react-client/src/hooks/screenShare.ts

```diff
@@ -23,8 +25,26 @@ export const useScreenShare = (): ScreenshareApi => {
     });
     const [video, audio] = getTracks(stream);
 
-    const addTrackPromises = [tsClient.addTrack(video)];
-    if (audio) addTrackPromises.push(tsClient.addTrack(audio));
+    const addTrackPromises = [
+      tsClient.addTrack(video).then(async (trackId) => {
```

Comment:
**question:** can't we just go with
```
tsClient.addTrack(video, { type: "screenShareVideo", paused: false })
```
?

Reply:
Yeah, you're right. I forgot that this is possible 😅